code/go/internal/validator/semantic/validate_kibana_filter_present.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 var (
+	errDashboardPanelWithoutFilter   = errors.New("at least one panel does not have a filter")
 	errDashboardWithFilterAndNoQuery = errors.New("saved query found, but no filter")
```

Comment:
Not related to this PR, but this error should not be renamed as `errDashboardWithQueryAndNoFilter` ?